docs/external-pipeline/results_in_rds.md

```diff
@@ -31,6 +31,7 @@ the service itself (messages with lower offset are skipped):
 CREATE TABLE report (
     org_id          INTEGER NOT NULL,
     cluster         VARCHAR NOT NULL UNIQUE,
+    account_number  INTEGER NOT NULL,
```

Comment:
hmm we don't need it to have there, WDYT? It will need some somplication on DB writer side, probably not worth to do it

Reply:
true, true